### PR TITLE
NDRS-412: Add Chainspec information to GetStatus feed

### PR DESCRIPTION
**Pull request by darthsiroftardis**
Related to JIRA ticket:
https://casperlabs.atlassian.net/browse/NDRS-413

### CODE REVIEW

File: node/src/types/status_feed.rs [hunk 1]
```diff
@@ -12,13 +12,20 @@ pub struct StatusFeed<I> {
     pub last_finalized_block: Option<Block>,
     /// The peer nodes which are connected to this node.
     pub peers: HashMap<I, SocketAddr>,
+    /// The chainspec info for this node.
+    pub chainspec_info: Option<ChainspecInfo>,
```
Comment (EdHastingsCasperLabs) [inline node/src/types/status_feed.rs:16]: nitpick: given that Option<...> was removed from the chainspec_loader side of this prior to opening the PR, you could also remove it here.

File: node/src/components/api_server.rs [hunk 2]
```diff
@@ -255,12 +256,14 @@ where
...
+                let (last_finalized_block, peers, chainspec_info) = join!(
                     effect_builder.get_last_finalized_block(),
-                    effect_builder.network_peers()
+                    effect_builder.network_peers(),
+                    effect_builder.get_chainspec_info()
                 );
-                let status_feed = StatusFeed::new(last_finalized_block, peers);
-                debug!("GetStatus --status_feed: {:?}", status_feed);
+                let status_feed =
+                    StatusFeed::new(last_finalized_block, peers, Some(chainspec_info));
```
Comment (EdHastingsCasperLabs) [inline node/src/components/api_server.rs:265]: see comment on StatusFeed

**Review by EdHastingsCasperLabs — APPROVED** [review]
Other than a nitpick, lgtm.

File: node/src/types/status_feed.rs (hunk 1 above)
Comment (darthsiroftardis) [reply node/src/types/status_feed.rs]: Updated it just `ChainspecInfo`

File: node/src/components/api_server.rs (hunk 2 above)
Comment (darthsiroftardis) [reply node/src/components/api_server.rs]: Same resolution as above

**Pull request merged**